src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+def test_sss_overrides__overriding_username_and_posix_attributes(client: Client, provider: GenericProvider):
+    """
+    :title: Locally overriding the name and POSIX attributes of a user
+    :setup:
+        1. Create POSIX user "user1" with standard POSIX attributes defined
+        2. Start SSSD
+    :steps:
+        1. Create local override for "user1" and name it "o-user1"
```

Comment:
Restart SSSD must be standalone step here, otherwise you will not be able to reproduce it.